### PR TITLE
Added support for eth, added mapping from rsk to eth.

### PR DESCRIPTION
**Pull request by valgaro**
Added initial files for eth support and a mapping in the index file to
use etc when the coin type is 'rsk'.
Ticket: BG-32

Signed-off-by: Gonzalo Alvarez <alvarez14@avaya.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,12 @@
     "elliptic": "^6.5.2",
     "libsodium-wrappers": "^0.7.6",
     "protobufjs": "^6.8.9",
-    "tronweb": "^2.7.2"
+    "tronweb": "^2.7.2",
+    "web3": "^1.2.6",
```
Comment (paszyrko) [inline package.json:46]: Add a comment saying that this will be removed later

File: src/coin/eth/enum.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+   */
+  Send,
+  /**
+   * This is the multi-sig initialization contract.
+   */
+  WalletInitialization,
+  /**
+   * This is the multi-sig contract configuration.
+   */
+  WalletConfiguration,
```
Comment (paszyrko) [inline src/coin/eth/enum.ts]: Remove this entry

File: src/coin/eth/signature.ts [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+    self._fields = [];
+
+    self.serialize = function serialize() {
+      return encode(self.raw);
+    };
+
+    fields.forEach((field, i) => {
+      self._fields.push(field.name);
+      /**
+       *
```
Comment (paszyrko) [inline src/coin/eth/signature.ts:240]: clean these comments

File: src/coin/eth/utils.ts [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+import keccak256 from 'keccak256';
+import BN from 'bn.js';
+import Web3 from 'web3';
+import _ from 'underscore';
+
+export type ByteArray = number[];
+const web3 = new Web3(rskDomain);
+
+/**
+ * Signs the transaction using the Eth elliptic curve
```
Comment (paszyrko) [inline src/coin/eth/utils.ts:17]: General comment. I think it is necessary to add documentation to the methods :(

File: src/coin/eth/enum.ts (hunk 2 above)
Comment (valgaro) [reply src/coin/eth/enum.ts]: Done

File: package.json (hunk 1 above)
Comment (valgaro) [reply package.json:46]: I can't comment the json but I will do it in utils.js

**Pull request merged**